clang/lib/Sema/SemaChecking.cpp

```diff
@@ -13846,6 +13880,37 @@ Sema::CheckReturnValExpr(Expr *RetValExp, QualType lhsType,
     CheckPPCMMAType(RetValExp->getType(), ReturnLoc);
 }
 
+/// Diagnose comparison to NAN or INFINITY in fast math modes.
+/// The comparison to NaN or INFINITY is always false in
+/// fast modes: float evaluation will not result in inf or nan.
+void Sema::CheckInfNaNFloatComparison(SourceLocation Loc, Expr *LHS, Expr *RHS,
```

Comment:
This line is 81 characters long.

Reply:
It's 79 chars long.

Reply:
Oh, sorry. I must have copied the "+ " when I checked it. It just looked longer than the surrounding text.